POC/py/parser.py

```diff
@@ -0,0 +1,306 @@
+import sys
+import re
+import os
+import json
+import asn1tools
+import shutil
+from collections import deque
+
+#accept multiple files
+#check and validate for well formness of asn1 file by compiling input using asn1 tools
+
+# take a file name as arg
+file_path=sys.argv[1]
+
+if file_path.endswith('.asn1'):
+    with open(file_path, 'r') as file:
+        data = file.read()
+        # print(data)
+else:
+    print("Invalid file type")
```

Comment:
make the error more descriptive, saying this something like code accepts ".asn1" types